### ISSUE TITLE
Timestamp container: bundle splitting work + 4 new services for Moment.js

### ISSUE THREAD

**Issue by j-pendlebury** [bbc/simorgh#1664]
**Is your feature request related to a problem? Please describe.**
#1566 is being split into two separate issues. This issue will handle bundle splitting (https://github.com/bbc/simorgh/issues/1602) and how we can add 4 extra Moment.js locales.

**Describe the solution you'd like**
Moment.js locales for Igbo, Yoruba, Thai, and Pidgin added to the bundle for each service.

Blocked by Bundle Splitting

**Testing notes**
[Tester to complete]

Dev insight: 

**Additional context**

**Comment by j-pendlebury** [comment]
No longer relevant
